includes/types.h

```diff
@@ -1,10 +1,38 @@
 #ifndef TYPES_H
 # define TYPES_H
 
-typedef struct 	s_minishell
+/* PARSING MACROS */
+# define SPACE ' '
+# define PIPE '|'
+# define ENV_VARIABLE '$'

```

Comment:
I forgot before sry, can it be ENV_VAR please please pleaseeeeeeeee

Reply:
haha. ok.